src/interfaces/stepper-interfaces/PeopleAndRoles/org-person-interface.ts

```diff
@@ -3,7 +3,7 @@ import { ActionTypes, IncorporatorTypes } from '@/enums'
 
 export interface OrgPersonIF {
   officer: {
-    id: number
+    id: string
```

Comment:
What I don't know is what happens if the API returns id as a number... Does it get cast to a string?

Reply:
(This would be for existing drafts.)

Reply:
(Or if the UI is deployed before the API.)

Reply:
Answer: it doesn't work well because some code does strict comparison between IDs.

So it's not backwards compatible.